app/main.py

```diff
@@ -1,8 +1,31 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str = "", age: int = 0) -> None:
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    people_dict = Person.people
+    for person in people:
+        person_name = person["name"]
+        person_age = person["age"]
+        spouse = "wife" if "wife" in list(person.keys()) else "husband"
+        spouse_name = person[spouse]
```

Comment:
you don't need to do this as you already have this data in people list (which is on function entry)